src/services/management/collector-history/collect-job/modules/JobTaskErrorList.vue

```diff
@@ -13,9 +13,23 @@
         >
             <template #col-format="{field: { name }, item, value}">
                 <template v-if="name === 'message'">
-                    <div class="error-message">
+                    <div v-if="value.length >= 80" class="error-message">
                         <pre class="content" :class="{ 'text-overflow': errorItemsCollapsibleList[item.sequence - 1] }">{{ value }}</pre>
-                        <p-collapsible-toggle :is-collapsed.sync="errorItemsCollapsibleList[item.sequence - 1]" />
+                        <div class="toggle-box">
+                            <p-collapsible-toggle v-if="value.length >= 80" :is-collapsed.sync="errorItemsCollapsibleList[item.sequence - 1]" />
+                        </div>
+                    </div>
+                    <div v-else :style="{whiteSpace: 'nowrap'}">
+                        <span>{{ value ? value : '--' }}</span>
+                    </div>
```

Comment:
```suggestion
                    <span v-else :style="{whiteSpace: 'nowrap'}">
                        {{ value ? value : '--' }}
                    </span>
```
태그가 불필요하게 겹겹이 많은 것 같아요.
혹시 위처럼 했을 때 문제가 생긴다면 무시해주세요.